include/autoware/mtr/agent.hpp

```diff
@@ -280,7 +280,7 @@ struct AgentData
     const std::vector<AgentHistory> & histories, const size_t ego_index,
     const std::vector<size_t> & target_indices, const std::vector<size_t> & label_ids,
     const std::vector<float> & timestamps)
-  : num_target_(target_indices.size()),
+  : num_target_(target_indices.size() + 1),  // NPC targets + Ego
```

Comment:
Currently, we've excluded EGO from the target indices intentionally:
https://github.com/ktro2828/autoware_mtr/blob/5a55bfab0373f2885b0d279ec0f734edec141846/src/node.cpp#L367

I think we don't have to update this line, but we need to update operations to extract target indices for EGO.
```suggestion
  : num_target_(target_indices.size()), 
```

Reply:
I agree. I fixed it back and I will look into adding the ego as a target. That was causing some crashes though so I need to investigate why